### DISCUSSION TITLE
Guess the artist 🖼️🎨🖌️

### DISCUSSION THREAD

**Discussion by ParanoidUser** [ParanoidUser/yolo#1237] [Galaxy Trivia] [answered]
![image](https://github.com/ParanoidUser/yolo/assets/5120290/3216a6b9-2120-4ab8-98c6-9295ed32a29b)

**Comment by peterbaccaro** [comment, marked as answer]
The painting you are probably referring to is the 1930s painting called American Gothic by the artist **Grant Wood**.

![image](https://github.com/ParanoidUser/yolo/assets/74666079/cd4f4293-f027-4bb0-b9ba-88e34a73bf65)
